src/domcoords.js

```diff
@@ -283,16 +283,18 @@ export function coordsAtPos(view, pos) {
   let {node, offset} = view.docView.domFromPos(pos)
 
   // These browsers support querying empty text ranges
-  if (node.nodeType == 3 && (browser.chrome || browser.gecko)) {
+  if (node.nodeType == 3 && browser.webkit) {
     let rect = singleRect(textRange(node, offset, offset), 0)
     // Firefox returns bad results (the position before the space)
     // when querying a position directly after line-broken
     // whitespace. Detect this situation and and kludge around it
     if (browser.gecko && offset && /\s/.test(node.nodeValue[offset - 1]) && offset < node.nodeValue.length) {
       let rectBefore = singleRect(textRange(node, offset - 1, offset - 1), -1)
-      if (Math.abs(rectBefore.left - rect.left) < 1 && rectBefore.top == rect.top) {
+      if (rectBefore.top == rect.top) {
```

Comment:
The check on the `left` property is dropped here. Is that intentional? Why?

Reply:
I was addressing only issue with `top` and `bottom` coordinates which in this combination didn't pass this condition. `Math.abs(rectBefore.left - rect.left)` on Firefox (70.0.1) was always approx 5px.
Do you know the reason why we need to check `left` property as well?

Reply:
I believe it is there to make sure the rect for the position before the target is an actual unbroken, cursor-shaped rectangle, but not entirely sure. I'll play around with this ab bit more to see what the rectangles actually look like.